app/renderer/components/config_menu.js

```diff
@@ -1,4 +1,7 @@
 "use strict";
+const app = require('electron').remote;
+const dialog = app.dialog;
```

Comment:
as a note for the future (**no need to change this now**) personally I like to add the main dependencies first, and any change at the bottom. These kind of agreements are simply for consistency and ease of reading.

So, here it would be (comments added to explain):
```js
const fs = require('fs'); // System built-in library, no dependency
const app = require('electron').remote; // External dependency
              // Space to show the difference between dependencies and our code
const AppStatus = require('../app_status');  // These are our own modules
const SwitchForm = require('./switch_form');
      // again, space to show the following are not imports
const dialog = app.dialog; // constant, not a dependency, but a variable/process
```
Each project has its own conventions on this, there is not a strict rule, I simply try to follow the convention explained in the example in this project